src/ReflectionNamedType.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace PhpDocumentGenerator;
+
+class ReflectionNamedType extends \ReflectionNamedType
```

Comment:
Please add a comment explaining why this class has been structured like that (I guess, because of the \ReflectionNamedType class inheritance)